content/en/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -37,71 +32,71 @@ restarts. Here is the configuration file for the Pod:
 
 1. Create the Pod:
 
-    ```shell
-    kubectl apply -f https://k8s.io/examples/pods/storage/redis.yaml
-    ```
+   ```shell
+   kubectl apply -f https://k8s.io/examples/pods/storage/redis.yaml
+   ```
 
 1. Verify that the Pod's Container is running, and then watch for changes to
-the Pod:
+   the Pod:
+
+   ```shell
+   kubectl get pod redis --watch
+   ```
 
-    ```shell
-    kubectl get pod redis --watch
-    ```
-    
-    The output looks like this:
+   The output looks like this:
 
-    ```shell
-    NAME      READY     STATUS    RESTARTS   AGE
-    redis     1/1       Running   0          13s
-    ```
+   ```shell
```

Comment:
(aside) We should also fix this line: the sample output is not a valid shell program.

I can see that a fix here is different to the intent of this PR.

Reply:
ok, noted. It shall be `none` or `console` 👍